homeassistant/components/netgear_lte/config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Config flow for Netgear LTE integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from aiohttp.cookiejar import CookieJar
+from eternalegypt import Error, Modem
+from eternalegypt.eternalegypt import Information
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DEFAULT_HOST, DOMAIN, MANUFACTURER
+
+
+class NetgearLTEFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Netgear LTE."""
+
+    async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
+        """Import a configuration from config.yaml."""
+        return await self.async_step_user(
+            user_input={
```

Comment:
Maybe we could verify the data like in the PR linked in the other comment

Reply:
Is it not already being verified using the user flow?